drivers/espi/espi_it8xxx2.c

```diff
@@ -153,6 +155,106 @@ static const struct ec2i_t pmc1_settings[] = {
 	{HOST_INDEX_LDA, 0x01},
 };
 
+#ifdef CONFIG_ESPI_PERIPHERAL_EC_HOST_CMD
+#define IT8XXX2_ESPI_HC_DATA_PORT_MSB \
+	((CONFIG_ESPI_PERIPHERAL_HOST_CMD_DATA_PORT_NUM >> 8) & 0xff)
+#define IT8XXX2_ESPI_HC_DATA_PORT_LSB \
+	(CONFIG_ESPI_PERIPHERAL_HOST_CMD_DATA_PORT_NUM & 0xff)
+#define IT8XXX2_ESPI_HC_CMD_PORT_MSB \
+	(((CONFIG_ESPI_PERIPHERAL_HOST_CMD_DATA_PORT_NUM + 4) >> 8) & 0xff)
+#define IT8XXX2_ESPI_HC_CMD_PORT_LSB \
+	((CONFIG_ESPI_PERIPHERAL_HOST_CMD_DATA_PORT_NUM + 4) & 0xff)
+static const struct ec2i_t pmc2_settings[] = {
+	/* Select logical device 12h(PM2 host command) */
+	{HOST_INDEX_LDN, LDN_PMC2},
+	/* I/O Port Base Address (data/command ports) */
+	{HOST_INDEX_IOBAD0_MSB, IT8XXX2_ESPI_HC_DATA_PORT_MSB},
+	{HOST_INDEX_IOBAD0_LSB, IT8XXX2_ESPI_HC_DATA_PORT_LSB},
+	{HOST_INDEX_IOBAD1_MSB, IT8XXX2_ESPI_HC_CMD_PORT_MSB},
+	{HOST_INDEX_IOBAD1_LSB, IT8XXX2_ESPI_HC_CMD_PORT_LSB},
+	/* Set IRQ=00h for logical device */
+	{HOST_INDEX_IRQNUMX, 0x00},
+	/* Enable logical device */
+	{HOST_INDEX_LDA, 0x01},
+};
+#endif
+
+#if defined(CONFIG_ESPI_PERIPHERAL_EC_HOST_CMD) || \
+	defined(CONFIG_ESPI_PERIPHERAL_ACPI_SHM_REGION)
+/* Host to RAM (H2RAM) memory mapping, 4K aligned */
+#define IT8XXX2_ESPI_H2RAM_POOL_SIZE 0x1000
+static uint8_t h2ram_pool[IT8XXX2_ESPI_H2RAM_POOL_SIZE]
+					__attribute__((section(".h2ram_pool")));
```

Comment:
From where do this 4k alignment requirement originates ?
I think this code should have a comment describing why there is such a requirement.

Reply:
Done.